doc/devel/documenting_mpl.rst

```diff
@@ -51,17 +52,86 @@ requirements that are needed to build the documentation. They are listed in
 
 * Sphinx>=1.3, !=1.5.0, !=1.6.4, !=1.7.3
 * colorspacious
-* IPython
+* IPython<7.0.0
```

Comment:
Needs to be clarified wrt #12583.

Reply:
This will get out of sync again in the future. Can we just refer to `doc-requirements.txt` instead of listing everything explicitly here again? Or [literalinclude](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude) if we want that information written out here.